File: src/main/java/br/com/solutis/desafio/security/LoginResponse.java
package br.com.solutis.desafio.security;

import br.com.solutis.desafio.domain.Usuario;

import java.util.Date;
import java.util.Map;

public class LoginResponse {
	
	private String token;
	Usuario usuario;
	public Date data;

	public String getToken() {
		return token;
	}

	public void setToken(String token) {
		this.token = token;
	}

	public Date getData() {
		return data;
	}

	public void setData(Date data) {
		this.data = data;
	}

	public Usuario getUsuario() {
		return usuario;
	}

	public void setUsuario(Usuario usuario) {
		this.usuario = usuario;
	}
}

File: src/main/Client/src/app/security/login-response.ts
import {Operador} from "../tributos/operador/operador";
import {UnidadeGestora} from "../tributos/unidadegestora/unidadegestora";
import {Entidade} from "../tributos/entidade/entidade";

export class LoginResponse{
  public usuario;
  public data: Date;
  public token: string;

    constructor() {
    }
}
